### PR TITLE
[MIG][18.0] fieldservice

### PR DESCRIPTION
**Pull request by EdgarRetes**

### CODE REVIEW

**Comment by EdgarRetes** [comment]
@max3903

**Comment by brian10048** [comment]
@EdgarRetes can you rebase this now that base_territory is merged?  I'll be glad to review

**Comment by EdgarRetes** [comment]
> @EdgarRetes can you rebase this now that base_territory is merged? I'll be glad to review
Can't find the cause of the error in the tests, got any ideas?

**Comment by brian10048** [comment]
@EdgarRetes I'm not seeing anything specifically wrong either.  I found [this change in odoo 18](https://github.com/OCA/OCB/commit/35abc30237d9286ec110e379ade1a94fa19bef0c) that may fix the issue if we implement the same change for multi-company domain rules

**Comment by EdgarRetes** [comment]
> @EdgarRetes I'm not seeing anything specifically wrong either. I found [this change in odoo 18](https://github.com/OCA/OCB/commit/35abc30237d9286ec110e379ade1a94fa19bef0c) that may fix the issue if we implement the same change for multi-company domain rules

That worked, thanks. i think it is ready for testing

**Comment by brian10048** [comment]
Awesome! Thanks for the work.  I will review and test it over the next day or so

File: fieldservice/views/fsm_template.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                    </notebook>
+                </sheet>
+            </form>
+        </field>
+    </record>
+    <record id="action_fsm_template" model="ir.actions.act_window">
+        <field name="name">Order Templates</field>
+        <field name="res_model">fsm.template</field>
+        <field name="view_id" ref="fsm_template_tree_view" />
+        <field name="view_mode">tree,form</field>
```
Comment (brian10048) [inline fieldservice/views/fsm_template.xml]:
```suggestion
        <field name="view_mode">list,form</field>
```

**Review by brian10048 — APPROVED** [review]
Code review and tested common functions on runboat

**Review by max3903 — APPROVED** [review]

**Pull request merged**